README.md

```diff
@@ -66,40 +47,24 @@ languages and cultures. See [i18n API](./guides/I18N-API.md) to get started.
-| Laurent Goderre      | [LaurentGoderre](https://github.com/LaurentGoderre)       | [LaurentGoderre](https://twitter.com/LaurentGoderre)     |
-| Łukasz Szewczak      | [lukaszewczak](https://github.com/lukaszewczak)           | [lukaszewczak](https://twitter.com/lukaszewczak)         |
-| Nick Schonning       | [nschonni](https://github.com/nschonni)                   | [nschonni](https://twitter.com/nschonni)                 |
-| Rachel White         | [rachelnicole](https://github.com/rachelnicole)           | [ohhoe](https://twitter.com/ohhoe)                       |
-| Raja Sekar           | [rajzshkr](https://github.com/rajzshkr)                   | [rajzshkr](https://twitter.com/rajzshkr)                 |
-| Richard Littauer     | [RichardLitt](https://github.com/RichardLitt)             | [richlitt](https://twitter.com/richlitt)                 |
-| Ryo Aramaki          | [ryo-a](https://github.com/ryo-a)                         | [geo_vitya](https://twitter.com/geo_vitya)               |
-| Ryuichi Okumura      | [okuryu](https://github.com/okuryu)                       | [okuryu](https://twitter.com/okuryu)                     |
-| Sam Yamashita        | [sotayamashita](https://github.com/sotayamashita)         | [sota0805](https://twitter.com/sota0805)                 |
-| Steven R. Loomis     | [srl295](https://github.com/srl295)                       | [srl295](https://twitter.com/srl295)                     |
-| Tamerlan Satualdypov | [onl1ner](https://github.com/onl1ner)                     | [onl1ner1](https://twitter.com/onl1ner1)                 |
-| Tiago Danin          | [TiagoDanin](https://github.com/TiagoDanin)               | [\_TiagoEDGE](https://twitter.com/_TiagoEDGE)            |
-| Vanessa Yuen         | [vanessayuenn](https://github.com/vanessayuenn)           | [vanessayuenn](https://twitter.com/vanessayuenn)         |
-| Volkan Nazmi Metin   | [Volem](https://github.com/Volem)                         | [volemnic](https://twitter.com/volemnic)                 |
-| Wexpo Lyu            | [laosb](https://github.com/laosb)                         | [it99p](https://twitter.com/it99p)                       |
-| Yuki Okushi          | [JohnTitor](https://github.com/JohnTitor)                 | [fr4ternite](https://twitter.com/fr4ternite)             |
-| Zeke Sikelianos      | [zeke](https://github.com/zeke)                           | [zeke](https://twitter.com/zeke)                         |
-
-## Joining
-
-If you're interested in joining this group or would like to leave a question or comment for its members, please [create an issue](https://github.com/nodejs/i18n/issues/new) or submit a pull request. Keep in mind, you don't have to be a member of this group to start translating. You can read more about contributing translations in the [Getting started](./guides/GETTING_STARTED.md) guide.
+- [amiller-gh](https://github.com/amiller-gh) - Adam Miller
```

Comment:
Could you check again if the names and links are the exactly same as the previous? The current seems broken.

Reply:
(I'm not sure if we really need to reformat the list though.)

Reply:
I updated it and used an easier to use format